lib/astral_plane/http_client/digital_ocean.rb

```diff
@@ -1,22 +1,24 @@
 class AstralPlane::DigitalOcean < AstralPlane::AbstractHttpClient
   BASE_URL = 'https://api.digitalocean.com/v2/'.freeze
-  # TODO: set this token in config
-  TOKEN = "token"
 
   def self.restart_server(droplet_id)
     body = '{"type":"reboot"}'
-    post(BASE_URL, "droplets/#{droplet_id}/actions", TOKEN, body)
+    post(BASE_URL, "droplets/#{droplet_id}/actions", token, body)
   end
 
   def self.create_server(body)
-    post(BASE_URL, "droplets", TOKEN, body)
+    post(BASE_URL, "droplets", token, body)
   end
 
   def self.delete_server(droplet_id)
-    delete(BASE_URL, "droplets/#{droplet_id}", TOKEN, nil)
+    delete(BASE_URL, "droplets/#{droplet_id}", token, nil)
   end
 
   def self.list_servers
-    get(BASE_URL, "droplets", TOKEN, nil)
+    get(BASE_URL, "droplets", token, nil)
+  end
+
+  def self.token
+    AstralPlane.config.digital_ocean_token
```

Comment:
@phereford  I initially tried to set this up as a constant like we had it originally, but when I attempted to spin up the gem in the console it threw a NoMethodError for `AstralPlane.config`. I'm thinking this is because it was trying to set the constant on this class before the metaprogramming had a chance to set the config method on `AstralPlane` since I'm able to call `config` just fine from the console after the gem builds.

Setting it as a class method instead solves that problem, but now I'm wondering if there's a better way of setting up the configurable module so that it runs before constants are set so we have the option of setting these config values as constants?

Reply:
@bradleyden I am not certain we can do constants now that we have gone down this thought path because it is technically not constant. Constants get evaluated on bootup (i BELIEVE), prior to your setting of the value in an initializer file (or other means). In this sense, I think this solution is best!

Lets move the `self.token` method into the Abstract HTTP Client as well with a `raise NotImplementedError`

Great job!!!

Reply:
Excellent point about the token not actually being a constant! To paraphrase Jeff Goldblum, I was so concerned with whether we could, I didn't stop to consider whether we _should_ 😁 

I will definitely add the method to the abstract class and get this merged today, thanks for taking a look!